optimum/onnxruntime/__init__.py

```diff
@@ -29,7 +29,11 @@
     ],
     "modeling_ort": [
         "ORTModel",
+        "ORTModelForAudioClassification",
+        "ORTModelForAudioFrameClassification",
+        "ORTModelForAudioXVector",
         "ORTModelForCustomTasks",
+        "ORTModelForCTC",
```

Comment:
Can you add them in the documentation .mdx as well?

Reply:
Done